.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,18 +4,18 @@ about: Suggest an idea for this project
 
 ---
 
-## :warning: Is your feature request related to a problem? Please describe.
-*A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]*
+## :warning: Is your feature request related to a problem? Please describe
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
 
-## :dart: Describe the solution you'd like
-*A clear and concise description of what you want to happen.*
+## :bulb: Describe the solution you'd like
+<!-- A clear and concise description of what you want to happen. -->
 
 ## :bar_chart: Describe alternatives you've considered
-*A clear and concise description of any alternative solutions or features you've considered.*
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
 
 ## :page_facing_up: Additional context
-*Add any other context or screenshots about the feature request here.*
+<!-- Add any other context or screenshots about the feature request here. -->
 
-## :pencil: Do you want to develop this feature yourself?
+## :raising_hand: Do you want to develop this feature yourself?
 - [ ] Yes
 - [X] No
```

Comment:
Maybe we can default to true? Or leave both blank?

Reply:
Both blank should work